File: lecture_002(Normal)/l_009_primenumbertillN.cpp
#include<iostream>
using namespace std;

void primenumbertilln() {
    cout<<"Enter the two numbers: ";
    int low; int high;                  
    cin>>low>>high;                                          //cin>>high>>low;
    cout<<"Prime Numbers between two numbers are: ";
    for(int n=low; n<=high; n++) {                           //for (int n=high; n>=low; n--)
        bool flag = false;
        for(int div =2; div*div <= n; div++) {
            if(n%div == 0) {
                flag = true;
                break;
            } 
        }
        if(flag == false) {
            cout<<n<<" "; 
        }
    }
}
int main(int args, char**argv) {
    primenumbertilln();

    return 0;
}

File: lecture_002(Normal)/l_009_primenumbertillN.java
import java.util.Scanner;
public class l_009_primenumbertillN {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.print("Enter two numbers: ");
        int low = scn.nextInt();                  //int high = scn,nextInt();
        int high = scn.nextInt();                 //int low = scn.nextInt();
        primeNumberTillN(low,high);
    }
    public static void primeNumberTillN(int low, int high) {
        System.out.print("Prime Numbers between two numbers are: ");
        for(int n=low; n<=high; n++) {                 //for (int n=high; n>=low; n--)
            int count = 0;
            for(int div=2; div*div<=n; div++) {
                if(n%div == 0) {
                    count++;
                    break;
                }
            }
            if(count != 1) {
                System.out.print(n+" ");
            }
        }
    }
}